math/src/unsigned_integer/element.rs

```diff
@@ -8,7 +8,6 @@ use crate::unsigned_integer::traits::IsUnsignedInteger;
 use std::fmt::Debug;
 
 pub type U384 = UnsignedInteger<6>;
-pub type U256 = UnsignedInteger<4>;
```

Comment:
Should this line be removed?

Reply:
Yeah, I had added it myself but then it turned out I did not need it so I removed it. However it turns out someone else had pushed the same line to main yesterday and I had missed that after merging. Fixed, thanks.